### DISCUSSION TITLE
Add small image / icon to the title panel

### DISCUSSION THREAD

**Discussion by jenspfahl-tgtg** [testingisdocumenting/znai#1113] [Ideas]
I can see there is a css class `.znai-documentation-logo` to display an icon in the upper left title panel but where can i define the actual image to be shown?

**Comment by MykolaGolubyev** [comment]
Try to put logo.png file next to toc file. Example here: https://github.com/testingisdocumenting/webtau/tree/master/webtau-docs/znai

**Reply by jenspfahl-tgtg** [reply]
It works, thank you!
